bionic.Dockerfile

```diff
@@ -0,0 +1,222 @@
+# ------------------------------------------------------------------------
+#
+# Copyright 2020 WSO2, Inc. (http://wso2.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+# ------------------------------------------------------------------------
+
+FROM ubuntu:bionic
+
+RUN \
+    apt-get update \
+    && apt-get install -y wget \
+    && apt-get install -y curl
+
+RUN \
+    apt-get -q update && apt-get install -y openssh-server software-properties-common git ant curl zip unzip xvfb dbus-x11 ttf-ancient-fonts \
```

Comment:
@Nirothipan shall we please break the `apt-get update` and `apt-get install` commands into multiple lines?

And please move package names into multiple lines in alphabetical order for clarity (see [example](https://github.com/wso2/docker-apim/blob/v3.1.0.3/dockerfiles/ubuntu/apim/Dockerfile#L58)).

Reply:
Please check for any other applicable instances within the PR, if any.

Reply:
@chirangaalwis Thanks for the review. Fixed.